src/main/resources/assets/nei/cfg/defaultbookmarkcontainers.cfg

```diff
@@ -0,0 +1,2 @@
+# Each line in this file should either be a comment (line starts with '#'), guiContainer class name.
```

Comment:
Please add a sentence at the top of this file explaining what this file is meant for, so that a non-GTNH pack author can understand it. There is no documentation on this anywhere, AFACIT, no even in the PR.